azurelinuxagent/common/conf.py

```diff
@@ -634,6 +634,14 @@ def get_enable_ga_versioning(conf=__conf__):
     return conf.get_switch("Debug.EnableGAVersioning", True)
 
 
+def get_enable_agent_update_in_dcr(conf=__conf__):
+    """
+    If True, the agent checks for updating the agent.
```

Comment:
can you clarify this comment? not sure it describes the purpose of the flag

Reply:
(assuming it is not just a temporary flag)

Reply:
if it is temporary, adding a comment REMOVE BEFORE MERGE TO DEVELOP or something like that may be a good idea